Files/Views/LayoutModes/GenericFileBrowser.xaml.cs

```diff
@@ -104,12 +104,12 @@ public override void SetSelectedItemOnUi(ListedItem item)
             AllView.SelectedItems.Add(item);
         }
 
-        public override void SetSelectedItemsOnUi(List<ListedItem> items)
+        public override void SetSelectedItemsOnUi(List<ListedItem> selectedItems)
         {
             ClearSelection();
-            foreach (ListedItem item in items)
+            foreach (ListedItem selectedItem in selectedItems)
             {
-                AllView.SelectedItems.Add(item);
+                AllView.SelectedItem = selectedItem;
```

Comment:
What purpose of this change? This broke file selection. File size is not more visible when changing view mode from `GridViewBrowser` to `GenericFileBrowser` and back.